com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ArchiveBuilder.java

```diff
@@ -228,7 +232,7 @@ public void write(RandomAccessFile archiveIndex, RandomAccessFile archiveData, P
                 archiveData.write(buffer, 0, buffer.length);
             }
 
-            manifestBuilder.addResourceEntry(normalisedPath, buffer, resourceEntryFlags);
+            manifestBuilder.addResourceEntry(normalisedPath, CryptographicOperations.toHashDigest(hashDigest), resourceEntryFlags);
```

Comment:
Use the already calculated hash digest instead if recalculating it in manifest builder based on the buffer (which would result in a digest mismatch in the case of Lua files)